sui-execution/latest/sui-move-natives/src/lib.rs

```diff
@@ -338,11 +355,77 @@ impl NativesCostTable {
                     .transfer_share_object_cost_base()
                     .into(),
             },
+            // tx_context
```

Comment:
Not sure about the meaning of this comment (is it cut short?)

Reply:
I can remove it but it was to highlight that the API below is for the `tx_context` and other part of this code do the same, so it was to replicate what is there already and to mark where that API is